### ISSUE TITLE
Missing form labels on several components 

### ISSUE THREAD

**Issue by philipbrembeck** [frontendnetwork/veganify#508]
Refer to https://wave.webaim.org/report#/https://vegancheck.me/

**Comment by philipbrembeck** [comment]
Already fixed
